### ISSUE TITLE
[Inquiry] VAEEncoder bug in KerasCV

### ISSUE THREAD

**Issue by LukeWood** [divamgupta/stable-diffusion-tensorflow#71]
Hello @divamgupta !  As you know, we ported the image encoder in https://github.com/keras-team/keras-cv from your library!  We've supported inpainting via a method on StableDiffusion, and we're seeing a very strange issue!  https://github.com/keras-team/keras-cv/issues/1172 is the github issue.

We've tried just passing images through the encoder/decoder and this padding shows up.

My question for you is: did you have this bug in your repo?  Did you submit a fix at some point?  Anything that you may know that may be helpful?

Thanks in advance for any guidance!  We appreciate your efforts a ton

**Comment by divamgupta** [comment]
You would need to use zero padding of 1 0 1 0 rather than 1 1 1 1

**Comment by LukeWood** [comment]
Thank you sir!
